var/spack/repos/builtin/packages/py-xdot/package.py

```diff
@@ -0,0 +1,83 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyXdot(PythonPackage):
+    """xdot.py is an interactive viewer for graphs written in Graphviz's
+    dot language."""
+
+    homepage = "https://github.com/jrfonseca/xdot.py"
+    url      = "https://github.com/jrfonseca/xdot.py/archive/0.9.tar.gz"
+    git      = "https://github.com/jrfonseca/xdot.py.git"
+
+    import_modules = ['xdot']
+
+    extends('python')
+
+    version('master', branch="master")
+    version('0.9.1', commit="0fa629166989576b05d509c7ef0329c0f7655190")
+    version('0.9', '19c78311d73b0f9ea059a6febf42eeea')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('python@3:')
+    depends_on('py-pygobject', type=('build', 'run'))
+    depends_on('py-pycairo', type=('build', 'run'))
+    depends_on('pango', type=('build', 'run'))
+    depends_on('atk', type=('build', 'run'))
+    depends_on('gdk-pixbuf', type=('build', 'run'))
+    depends_on('gtkplus', type=('build', 'run'))
+
+    @run_after('install')
+    def post_install(self):
+        spec = self.spec
+        repo_paths = '%s:%s:%s:%s' % (
+            join_path(spec['pango'].prefix.lib, 'girepository-1.0'),
+            join_path(spec['atk'].prefix.lib64, 'girepository-1.0'),
+            join_path(spec['gdk-pixbuf'].prefix.lib, 'girepository-1.0'),
+            join_path(spec['gtkplus'].prefix.lib, 'girepository-1.0'))
```

Comment:
These should probably be accessed via `spec['...'].libs` as `lib` vs `lib64` may depend on OS and whether it was externally installed or not.

Reply:
@adamjstewart  all 4 of those packages only produce versioned libraries (i.e., libgtk-3.so and libpango-1.0.so, etc.). Any way to work around this or should I leave the prefix.lib[64] specifications?

Reply:
Can we also postpone this to a following PR?

Reply:
The join_path calls are specific to the xdot package, so I'm hoping we can keep this in this PR. I'm not sure whether other packages would need to do something similar.

Regarding prefix.lib and prefix.lib64, yes, I propose that a separate issue be raised to do a better job finding versioned .so files.